pkg/reserved_resource/reserved_resource.go

```diff
@@ -0,0 +1,50 @@
+package reserved_resource
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift-online/ocm-support-cli/pkg/types"
+)
+
+type ReservedResource struct {
+	types.Meta
+	AvailabilityZoneType      string
+	BillingMarketplaceAccount string
```

Comment:
The field BillingMarketplaceAccount will be empty most of the time. We can use omitempty.